app/src/main/java/com/asfoundation/wallet/ui/backup/BackupCreationPresenter.kt

```diff
@@ -91,6 +100,14 @@ class BackupCreationPresenter(
   private fun handleDialogCancelClick() {
     disposables.add(view.getDialogCancelClick()
         .doOnNext { view.closeDialog() }
+        .doOnNext {
+          walletEventSender.sendWalletSaveFileEvent(WalletAnalytics.ACTION_CANCEL,
+              WalletAnalytics.STATUS_FAIL)
```

Comment:
Is it suppose to be Status_fail? the user cancels it.
Also, why not doing it on the previous doOnNext?